zerver/lib/import_realm.py

```diff
@@ -539,6 +539,15 @@ def do_import_realm(import_dir: Path, subdomain: str) -> Realm:
     # Import zerver_message and zerver_usermessage
     import_message_data(import_dir)
 
+    re_map_foreign_keys(data, 'zerver_reaction', 'message', related_table="message")
+    re_map_foreign_keys(data, 'zerver_reaction', 'user_profile', related_table="user_profile")
+    for reaction in data['zerver_reaction']:
+        if reaction['reaction_type'] == Reaction.REALM_EMOJI:
+            re_map_foreign_keys(data, 'zerver_reaction', 'emoji_code',
+                                related_table="realmemoji", id_field=True)
```

Comment:
Does `re_map_foreign_keys` do the right thing here?  It loops through the whole data table, not just the ones with that reaction type... I think we may need an additional argument to it for this application (probably a filter_function that one can pass in that determines which rows get remapped)

Reply:
Might make sense to put the commit fixing this on the bottom of the branch, since we have a few other issues here that may take more round-trips